pgbot/commands/base.py

```diff
@@ -160,7 +161,18 @@ async def cast_basic_arg(self, anno: str, arg: Any) -> Any:
             raise ValueError()
 
         elif isinstance(arg, str):
-            if anno in ["CodeBlock", "String", "datetime.datetime", "datetime"]:
+            if anno in ["CodeBlock", "String"]:
+                raise ValueError()
+
+            elif anno in ["datetime.datetime", "datetime"]:
+                if not arg.startswith("<t:") or not arg.endswith(">"):
+                    raise ValueError()
+
+                timestamp = re.search(r"\d+", arg)
+                if timestamp is not None:
+                    timestamp = float(arg[timestamp.start() : timestamp.end()])
+                    return datetime.datetime.fromtimestamp(timestamp)
```

Comment:
Here instead of `datetime.datetime.fromtimestamp`, we need to use `datetime.datetime.utcfromtimestamp`, because the former returns naive datetime object in the local timezone while the latter returns naive datetime object in UTC timezone